server/chat-commands/room-settings.ts

```diff
@@ -668,10 +668,8 @@ export const commands: ChatCommands = {
 		const targetRoom = Rooms.search(target);
 		if (!targetRoom) throw new Error(`Error in room creation.`);
 		if (cmd === 'makeprivatechatroom') {
-			targetRoom.settings.isPrivate = true;
 			if (!targetRoom.persist) throw new Error(`Private chat room created without settings.`);
```

Comment:
Does this even need to be thrown? This just using `/makeprivatechatroom` on a groupchat/battle, right?

Reply:
Looks like it. This shouldn't be necessary from what it seems, i agree.

Reply:
After some testing, this doesn't look like a problem, but it's definitely pretty confusing code.